docs/source/concepts/ledger-model/ledger-structure.rst

```diff
@@ -220,7 +218,7 @@ The following EBNF grammar summarizes the structure of commits and ledgers:
    Ledger   ::= Commit*
 
 A Daml ledger thus represents the full history of all actions taken by
-parties.\ [#ledger-vs-journal]_ Since the ledger is a sequence (of dependent actions), it induces an
+parties.\ [#ledger-vs-journal]_ Since the ledger has a sequence (of dependent actions), it induces an
```

Comment:
I preferred the old phrasing here. From an abstract pov (and we are talking about the ledger model here). Saying the the whole ledger is a sequence of dependent actions seems fine.

Reply:
yep, you're right, I got confused for a bit. reverting